src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1163,6 +1172,8 @@ internal TextLoader(IHostEnvironment env, Options options = null, IMultiStreamSo
             if (options.AllowQuoting && options.ReadMultilines)
                 _flags |= OptionFlags.ReadMultilines;
 
+            _escapeChar = options.EscapeChar;
+
```

Comment:
I need to check if _escapeChar is found in the separators and thow in that case.

Reply:
Do you plan to do it? :-)

---
In reply to: [428317215](https://github.com/dotnet/machinelearning/pull/5147#discussion_r428317215) [](ancestors = 428317215)

Reply:
Yeah, of course... I'm just waiting to receive more comments to address to sync this.

---
In reply to: [428359005](https://github.com/dotnet/machinelearning/pull/5147#discussion_r428359005) [](ancestors = 428359005,428317215)